db/users/create.go

```diff
@@ -0,0 +1,110 @@
+package users
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"golang.org/x/term"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/io"
+	"github.com/Scalingo/go-scalingo/v6"
+	scErrors "github.com/Scalingo/go-utils/errors/v2"
+	"github.com/Scalingo/gopassword"
+)
+
+func CreateUser(ctx context.Context, app, addonUUID, username string, readonly bool) error {
+	isSupported, err := isDatabaseHandlesUserManagement(ctx, app, addonUUID)
+	if err != nil {
+		return scErrors.Wrap(ctx, err, "get user management information")
+	}
+
+	if !isSupported {
+		io.Error(ErrDatabaseNotSupportUserManagement)
+		return nil
+	}
+
+	if usernameValidation, ok := isUsernameValid(username); !ok {
+		io.Error(usernameValidation)
+		return nil
+	}
+
+	password, confirmedPassword, err := askForPassword(ctx)
+	if err != nil {
+		return scErrors.Wrap(ctx, err, "ask for password")
+	}
+
+	passwordValidation, ok := isPasswordValid(password, confirmedPassword)
+	if !ok && password != "" {
+		io.Error(passwordValidation)
+		return nil
+	}
+
+	isPasswordGenerated := false
```

Comment:
**issue (blocking?):** the password generation logic is made backend sided. do we want to handle it on the client side?

Reply:
well that could have remove some complexity client sided, but this isn't really an issue and is not blocking, do as your heart wishes